cardano-testnet/src/Testnet/Conf.hs

```diff
@@ -24,21 +21,12 @@ newtype YamlFilePath = YamlFilePath
 
 data Conf = Conf
   { tempAbsPath :: FilePath
```

Comment:
Could we perhaps change the type of this to `TmpAbsolutePath`?

Reply:
Yes maybe do it in a followup PR ?

Reply:
Sure!